h/api/storage.py

```diff
@@ -37,10 +36,13 @@ def fetch_annotation(id):
     :returns: the annotation, if found, or None.
     :rtype: dict, NoneType
     """
-    return models.Annotation.fetch(id)
+    if _postgres_enabled(request):
```

Comment:
Being able to mock it out easily in test without affecting other features (as we sometimes do in tests) is the reason why I did it as a private method.